src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/TemplateInfo.cs

```diff
@@ -65,26 +65,31 @@ public bool AddVisited(object value)
             return _visitedObjects.Add(value);
         }
 
+        /// <summary>
+        /// Returns the full HTML element name for the specified <paramref name="partialFieldName"/>.
+        /// </summary>
+        /// <param name="partialFieldName">Expression name, relative to the current model.</param>
+        /// <returns>Fully-qualified expression name for <paramref name="partialFieldName"/>.</returns>
         public string GetFullHtmlFieldName(string partialFieldName)
```

Comment:
Changes here are just about added doc comments and consistency w/ the `NameAndIdProvider` version.